test/functional/legacy/038_virtual_replace_spec.lua

```diff
@@ -0,0 +1,56 @@
+-- Test Virtual replace mode.
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('Virtual replace mode', function()
+  setup(clear)
+
+  it('is working', function()
+    -- Make sure that backspace works, no matter what termcap is used.
+    execute('set t_kD=x7f t_kb=x08')
```

Comment:
This line contains two actual control characters `^V`. Our helpers support angle bracket notation, so you can replace this line with 

    execute('set t_kD=<C-v>x7f t_kb=<C-v>x08')